### ISSUE TITLE
Proposal 0001 - Reply Gating & Thread Moderation

### ISSUE THREAD

**Issue by Bossett** [bluesky-social/proposals#22]
I sat down to get my thoughts in order about community management and how I wanted to respond to the three proposals. That turned into a longer document available at https://bossett.io/bluesky-on-community-trust-safety/ which more clearly details my thoughts. This is an except from that that I've edited to make it stand-alone and not have a bunch of issues all conflated. I would appreciate any attention/feedback on that document (but not here - maybe at https://bsky.app/profile/bossett.bsky.social).

100% my own thoughts here - I've expressed it as 'recommendation' but only so that there's a strawman to argue the toss about.

## Reply Gating
Reply gating (to only people I follow, or only people on a list, or only people not on a list), is a good idea. This allows for targeted communication controlled by the end user, and adds a barrier to harassment.

However, at a cultural level, too many gated posts to an overly-limited audience will be a major impediment to inclusion. And using lists moderated by others for gating has similar drawbacks to overall trusted list management.

We have seen on other sites that quoting and screenshots will be used to attack people over gated comments, which implies that in use this will serve to set some guardrails, but not be a panacea.

### Recommendation
Reply gating is probably a net good. Should they become standard community practice, the system should be revisited either at a platform level or at a community level.

## Thread Moderation
Ultimately any manipulati0n of threads and reply chains comes down to one question: Who owns the reply content under a post?

Allowing deletion is likely the most straightforward answer to dealing with harassment online. If someone is creepy in replies, simply delete it. However, this kind of curation brings a significant downside: if the reply is correcting misinformation, or making a remark that goes against the initial poster's malicious intent - that poster has total control of the narrative.

Imagine a scenario where false state propaganda is posted, and every contrary reply is deleted. Even within a relatively small network, a post with 50 affirming replies is very powerful at pushing a message especially if the presence of deleted activity is completely 'behind the scenes'.

Reply hiding addresses this a little, but creates an 'attractive nuisance' situation where hidden replies are available to end users and there is a temptation to see 'what they are hiding'. In this instance, it may help to deal with creeps in replies, but not necessarily address misinformation that is posted to replies - instead feeding a suppression narrative.

Neither option is ideal, but reply hiding seems less bad because most users will see a reply via existing trust relationships (i.e. followers, etc.) and it's reasonable to assume that they will respect a poster's wishes. (At least, reasonable compared to the average visitor.)

Allowing threads to be locked can be treated similarly to reply gating in terms of moderation - where a reply is simply gated-off from all users. As a tool for 'settling' a thread, or just opting out as a poster, I think they're a good idea.

### Recommendation
Allow reply hiding, but not deletion. Do thread locking.

**Comment by daira** [comment]
(Not sure whether this is the right place to put this; I couldn't find a better place.)

If I understand correctly, if the initiator of a thread blocks a user then currently no-one will see that user's replies in the thread, *nor will they see that a reply has been hidden*. That's a problem because if, say, you reply to a troll and then the thread initiator blocks the troll, it will look as though you were replying to the parent of the troll's skeet.

Example where this problem is discussed: https://bsky.app/profile/themilfmag.net/post/3k7t7ot6e2726

**Comment by Bossett** [comment]
That one is probably a new issue, but it's the same impact - blocking has the 'deletion' effect and we're already seeing people use it to 'steer a narrative', or seeing things steered unintentionally.

The current set up is pretty bad, in a 'manufacture consensus' kind of way.

**Comment by daira** [comment]
I agree there's a more general issue in principle of "steering the narrative", but the immediate problem that I described is just an easily fixable UI bug: when a reply is preceded by skeets that are not being shown, say so in the UI.

**Comment by Bossett** [comment]
I was saying the immediate problem may be a new issue - i.e. not re: reply gating or thread locking. Possibly at https://github.com/bluesky-social/social-app/issues ? Not sure that the development team actively monitors here for things to include short term.
